### PR TITLE
Feat/pointer active states

### PR DESCRIPTION
**Pull request by thestonefox**

### CODE REVIEW

**Comment by Backer-Sultan** [comment]
@thestonefox 
A bug regarding this commit -> 1f0c66a9a86996ce4d65771af7c59dfe123ece50
When a pointer direction indicator is issigned to a bezier pointer renderer script, the bezier pointer renderer doesn't render the color to the cursor nor to the pointer direction indicator.
When no pointer direction indicator is assigned, the cursor gets the color correctly.

**Comment by thestonefox** [comment]
@Backer-Sultan thanks i'll take a look in a bit

**Comment by Backer-Sultan** [comment]
@thestonefox The pointer direction indicator doesn't rotate anymore too.

File: Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs [hunk 1]
```diff
@@ -72,8 +83,30 @@ public virtual Quaternion GetRotation()
...
         }
 
+        /// <summary>
+        /// The SetMaterialColor method sets the current material colour on the direction indicator.
+        /// </summary>
+        /// <param name="color">The colour to update the direction indicatormaterial to.</param>
+        /// <param name="validity">Determines if the colour being set is based from a valid location or invalid location.</param>
+        public virtual void SetMaterialColor(Color color, bool validity)
+        {
+            validLocation.SetActive(validity);
```
Comment (Backer-Sultan) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: This line Throws an NullReferenceException Whe the pointer is activated.

File: Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs [hunk 2]
```diff
@@ -72,8 +83,30 @@ public virtual Quaternion GetRotation()
...
+                for (int i = 0; i < renderers.Length; i++)
+                {
+                    renderers[i].material.color = color;
+                }
+            }
+        }
+
         protected virtual void Awake()
         {
+            validLocation = transform.Find("ValidLocation").gameObject;
```
Comment (Backer-Sultan) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: Line 108 throws NullReferenceException.

Comment (thestonefox) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: Are you actually checking out this pull request or just trying the code in your own scene?

File: Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs (hunk 1 above)
Comment (thestonefox) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: Are you actually checking out this pull request or just trying the code in your own scene?

Comment (Backer-Sultan) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: My fault! 😅 
I was trying the code on my own scene. And I didn't notice the change in the structure.
It's working fine!

File: Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs (hunk 2 above)
Comment (Backer-Sultan) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: My fault! 😅 
I was trying the code on my own scene. And I didn't notice the change in the structure.
It's working fine!

Comment (thestonefox) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: cool, no worries :) do you think this pull request is ready to merge?

Comment (Backer-Sultan) [inline Assets/VRTK/Prefabs/Resources/Scripts/VRTK_PointerDirectionIndicator.cs]: Yes, I do! I've pulled it already to my project and everything is working as expected.

**Pull request merged**